sdk/Lusid.Sdk.Tests/tutorials/Ibor/Valuation.cs

```diff
@@ -518,30 +513,5 @@ public void MultiDateValuationOfAnInstrumentPortfolio()
                 Assert.That(result[HoldingPvKey], Is.GreaterThanOrEqualTo(0));
```

Comment:
This doesn't really help a user understand how the instruments are working in the valuation engine. What is the intent of this example? It would be better to bring back relevant metrics for each asset class and also show what is(n't) returned where its not applicable

Reply:
What do you want to see that shows how the instruments work in the VE?
This example currently shows we can price successfully the instrument and the price is positive.
The new framework is using a set list of metrics. Which ones do you have in mind? we should be able to price all instruments. We are not currently doing specialised metrics e.g. asking for a bond duration - maybe for future work but there's enough to get working with just pv etc.